src/time-slot-bar/time-slot-bar.tsx

```diff
@@ -116,15 +119,31 @@ export const TimeSlotBar = ({
         const startTimeFormatted = dayjs(startTime, "HH:mm");
         const endTimeFormatted = dayjs(endTime, "HH:mm");
 
+        const isStartHourEven = startTimeFormatted.hour() % 2 === 0;
+
         for (
             let currentTime = startTimeFormatted;
             currentTime.isBefore(endTimeFormatted);
             currentTime = currentTime.add(CELL_DURATION, "minute")
         ) {
             const isHour = currentTime.minute() === 0;
+
+            const isAlternateHour =
```

Comment:
i find this logic abit hard to follow. Instead of using a const, could we extract it into a function instead? So `$isLongMarker` prop to use the function to get a boolean value

Reply:
Extracted to a separate helper function